tests/python/pants_test/base_test.py

```diff
@@ -483,9 +483,16 @@ def warnings(self):
       return self._messages_for_level('WARNING')
 
   @contextmanager
-  def captured_logging(self):
+  def captured_logging(self, level=None):
```

Comment:
could make more sense to default to `level=logging.INFO` here vs `NOTSET` below?

Reply:
I'm always maybe unjustifiably worried about someone then saying `captured_logging(level=None)` so I ~always defer the conversion. This probably negatively impacts pydoc though.